docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -5,26 +5,37 @@ HTTP connection management
 
 HTTP is such a critical component of modern service oriented architectures that Envoy implements a
 large amount of HTTP specific functionality. Envoy has a built in network level filter called the
-:ref:`HTTP connection manager <config_http_conn_man>`. This filter translates raw bytes into HTTP
-level messages and events (e.g., headers received, body data received, trailers received, etc.). It
-also handles functionality common to all HTTP connections and requests such as :ref:`access logging
-<arch_overview_access_logs>`, :ref:`request ID generation and tracing <arch_overview_tracing>`,
+:ref:`HTTP connection manager <config_http_conn_man>`.
+
+This filter translates raw bytes into HTTP level messages and events (e.g., headers received,
+body data received, trailers received, etc.).
+
+It also handles functionality common to all HTTP connections and requests such as
+:ref:`access logging <arch_overview_access_logs>`, :ref:`request ID generation and tracing <arch_overview_tracing>`,
 :ref:`request/response header manipulation <config_http_conn_man_headers>`, :ref:`route table
 <arch_overview_http_routing>` management, and :ref:`statistics <config_http_conn_man_stats>`.
 
-HTTP connection manager :ref:`configuration <config_http_conn_man>`.
+.. tip::
+   See the HTTP connection manager :ref:`configuration <config_http_conn_man>` and
+   :ref:`protobuf <envoy_v3_api_file_envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto>`
+   sections for reference documentation.
 
 .. _arch_overview_http_protocols:
 
 HTTP protocols
 --------------
 
-Envoy’s HTTP connection manager has native support for HTTP/1.1, WebSockets, HTTP/2 and HTTP/3. It does not support
-SPDY. Envoy’s HTTP support was designed to first and foremost be an HTTP/2 multiplexing proxy.
-Internally, HTTP/2 terminology is used to describe system components. For example, an HTTP request
-and response take place on a *stream*. A codec API is used to translate from different wire
-protocols into a protocol agnostic form for streams, requests, responses, etc. In the case of
-HTTP/1.1, the codec translates the serial/pipelining capabilities of the protocol into something
+Envoy’s HTTP connection manager has native support for **HTTP/1.1**, **WebSockets**, **HTTP/2** and **HTTP/3**.
```

Comment:
Hm, this already existed but it's weird to say H1, websockets, H2, H3.
Maybe H1, H2,and H3.  HTTP support includes websocket support over all 3 supported versions of HTTP?

Reply:
cool, lets just remove the WebSockets bit

Reply:
I think it's worth calling out, just not in the middle of the HTTPs

Reply:
ah ok - ill add it back as a general inclusion somehow